### ISSUE TITLE
pygments highlighting sometimes has bad contrast

### ISSUE THREAD

**Issue by not-my-profile** [pradyunsg/furo#82]
This can be observed at https://pip.pypa.io/en/stable/reference/pip_show/:

![image](https://user-images.githubusercontent.com/73739153/107882402-5f493280-6ee9-11eb-9b50-481cc96baa37.png)

A contrast of 3.34 does not meet the WCAG standards. (This is also reported by your browser if you right click on the text with the bad contrast, click Inspect and then click on the color).

AA conformance could be achieved with the following CSS rule:
```css
.highlight .go {
    color: #727272;
}
```

**Comment by pradyunsg** [comment]
Thanks for filing this issue!

This isn’t directly controlled by Furo, but is coming from pygments. I don’t see any there’s built-in alternative pygments theme that Furo should use that is much better, and this is customisable by the specific projects as well. I’m not sure what’s actionable here.

**Comment by not-my-profile** [comment]
Ah thanks, I thought this was caused by the Furo background color but apparently many pygments styles have bad contrasts for some token types. I opened an issue at Pygments: pygments/pygments#1718. In my report there you can also see that there are some built-in styles that have better contrasts but if Pygments improves the contrast of their default style, I don't think you need to take any action.

**Comment by pradyunsg** [comment]
Thanks for taking this upstream! I've subscribed to that and let's see what pygments' maintainers say.

I'll close this for now, since I don't see anything directly actionable here. :)
